### PR TITLE
[MIG] website_salesuggest_create_account: basic mig from v10 to v11

### PR DESCRIPTION
**Pull request by oscarolar**

### CODE REVIEW

File: website_sale_suggest_create_account/__manifest__.py [hunk 1]
```diff
@@ -4,11 +4,12 @@
 {
     "name": "Suggest to create user account when buying",
     "summary": "Suggest users to create an account when buying in the website",
-    "version": "10.0.1.1.0",
+    "version": "11.0.0.0.0",
```
Comment (yelizariev) [inline website_sale_suggest_create_account/__manifest__.py]: It should be 11.0.1.0.0

**Review by yelizariev — COMMENTED** [review]
Update version. The rest updated code seems ok (I didn't try on runbot)

**Comment by oscarolar** [comment]
@pedrobaeza @yelizariev all comments done.

**Review by moylop260 — APPROVED** [review]

**Comment by oscarolar** [comment]
@pedrobaeza I think this is ready to review

File: website_sale_suggest_create_account/__manifest__.py (hunk 1 above)
Comment (pedrobaeza) [reply website_sale_suggest_create_account/__manifest__.py]: This comment is still unattended

File: website_sale_suggest_create_account/__manifest__.py [hunk 2]
```diff
@@ -4,11 +4,12 @@
...
     "category": "Website",
     "website": "https://www.tecnativa.com",
     "author": "Tecnativa, "
               "LasLabs, "
+              "Vauxoo, "
```
Comment (pedrobaeza) [inline website_sale_suggest_create_account/__manifest__.py]: Changes done doesn't deserve co-authorship IMO. If you would add tests...

**Review by yajo — APPROVED** [review]
Squash migration commits to merge please

**Comment by moylop260** [comment]
@Yajo 
Are you sure that he should squash migration commits?

FYI https://github.com/OCA/maintainer-tools/wiki/Migration-to-version-11.0#technical-method-to-migrate-a-module-from-100-to-110-branch

File: website_sale_suggest_create_account/__manifest__.py (hunk 2 above)
Comment (moylop260) [reply website_sale_suggest_create_account/__manifest__.py]: I'm not sure what is the OCA criteria to add a co-authorship.

Could you send us the guideline about this matter, please?

Comment (max3903) [reply website_sale_suggest_create_account/__manifest__.py]: @moylop260 There is no guideline yet and today this is open to interpretation.

My opinion is:
- Migration, bugfix, translation, documentation --> Contributors
- Development, Refactoring, Co-development, Tests development --> Autors

Comment (pedrobaeza) [reply website_sale_suggest_create_account/__manifest__.py]: We can put in the middle migrations that require serious efforts for making it work, but this one is not clearly one. That's why I say that if you add tests, then authorship won't be discussed.

**Comment by yajo** [comment]
By "migration commits" I mean commits related to v10-v11 migration, not older history. Sorry, I didn't explain myself 😅

**Comment by oscarolar** [comment]
@Yajo done

**Pull request merged**